CHANGES

```diff
@@ -57,6 +58,7 @@ Bugs fixed
   + or ^) are used as keystrokes
 * #8629: html: A type warning for html_use_opensearch is shown twice
 * #8665: html theme: Could not override globaltoc_maxdepth in theme.conf
+* #4304: linkcheck: Do not check the availability of the same URL repeatedly
```

Comment:
This entry is misleading. It implies the availability was being checked repeatedly, when in most cases it wasn’t.

Perhaps:

Fix race condition that could lead to checking the availability of the same URL twice

Reply:
Indeed!